styleguide/derek/pattern-components/resourceCenter-resource/refresh/_resourceCenter-resource.scss

```diff
@@ -0,0 +1,141 @@
+// Resource Icon Color
+@mixin resource-refresh($class, $color) {
```

Comment:
`$color` here implies you're always passing in a color.  Rename this to `$background` as that is more indicative of what you're actually doing.